### DISCUSSION TITLE
How to config attribute-based access control (ABAC)  in keycloak UI?

### DISCUSSION THREAD

**Discussion by owenJiao** [keycloak/keycloak#37260] [Q&A]
### Description

1: The keycloak version I used is 18.0.0.
2: I found out keycloak supports attribute-based access control (ABAC)  , But I can't config this policy in keycloak UI,  I found from the documentation that the recommended way is to deploy JS code via JARs,  too much inconvenient. So my question is whether there is a better way to config ABAC policy?

**Comment by igor-zp** [comment]
Good question!
